srcpkgs/python3-GitPython/template

```diff
@@ -32,4 +32,7 @@ pre_check() {
 
 	git config --global user.email "void@example.com"
```

Comment:
would it be possible to use `git config --local` instead? would be good to not dirty the masterdir with this

Reply:
probably? I've changed it locally and am running the tests now. If it works, I'll commit and push that.

Reply:
seems like at least the new line added here has to be added globally.

Reply:
ok, doesn't seem too bad i guess

Reply:
well, nevermind then. let's put it back. maybe `rm ~/.gitconfig` or equivalent at the end?

Reply:
committed.